### PR TITLE
DR-2296 Don't check for existence of unused properties

### PR DESCRIPTION
**Pull request by nmalfroy**
...since we removed these properties

### CODE REVIEW

**Review by snf2ye — APPROVED** [review]
Thank you!

**Review by fboulnois — APPROVED** [review]

File: datarepo-clienttests/src/main/java/scripts/deploymentscripts/ModularHelmChart.java
Comment (pshapiro4broad) [inline datarepo-clienttests/src/main/java/scripts/deploymentscripts/ModularHelmChart.java:226]:
```suggestion
        List.of(
```

**Review by pshapiro4broad — APPROVED** [review]

File: datarepo-clienttests/src/main/java/scripts/deploymentscripts/ModularHelmChart.java
Comment (nmalfroy) [reply datarepo-clienttests/src/main/java/scripts/deploymentscripts/ModularHelmChart.java:226]: I'm gonna do a separate pass on all of the Arrays.asList things.  I don't want to hold off merging this to unblock the tests

**Pull request merged**
